### ISSUE TITLE
deinitialize() and deallocate() causing Compiler Error in SHA256.swift

### ISSUE THREAD

**Issue by SerialForBreakfast** [agens-no/EllipticCurveKeyPair#42]
Installed via cocoapods and got these errors when building:

'deinitialize()' is unavailable: the default argument to deinitialize(count:) has been removed, please specify the count explicitly
'deallocate(capacity:)' is unavailable: Swift currently only supports freeing entire heap blocks, use deallocate() instead

'deinitialize()' was obsoleted in Swift 5.0
'deallocate(capacity:)' was obsoleted in Swift 5.0

**Comment by SerialForBreakfast** [comment]
Works fine when using Swift 4.2 in the Build Settings.

**Comment by hfossli** [comment]
Yes, that's right. Fixed in master. If you point directly to commit or branch it works in swift 5

**Comment by hfossli** [comment]
Thanks for taking the time to report this. Keeping it open since I forgot to publish new release

**Comment by barrcodes** [comment]
Still no release, it's been a month

**Comment by hfossli** [comment]
I am so sorry. The past months have been crazy hectic both on a personal level and professional level. I am triaging every day in order to keep most important things floating. If someone wants push access to fix what's needed I'll be happy to give. Otherwise I will come back to this issue asap (hopefully this week). Thanks for being patient. 🙌

**Comment by fmessina** [comment]
Hey @hfossli, first of all thanks for this library!
Any hope to have the Swift5 fix released? 🙏🏼
